Sources/SwiftSyntaxBuilder/generated/BuildableCollectionNodes.swift

```diff
@@ -14,126 +14,126 @@
 
 import SwiftSyntax
 
-/// `AccessorList` represents a collection of `AccessorDeclSyntax`
+/// `AccessorListSyntax` represents a collection of `AccessorDeclSyntax`
```

Comment:
These docs don't seem particularly useful to me. I'd expect them on the type itself rather than the extension (and even there... fairly dubious about their value).

Reply:
I won’t argue with you about that. I hope that we’ll write proper documentation soon and then these will just automatically go away.